plugins/providers/bigquery/client.go

```diff
@@ -327,6 +306,34 @@ func (c *bigQueryClient) ListAccess(ctx context.Context, resources []*domain.Res
 	return access, nil
 }
 
+func (c *bigQueryClient) getGrantAbleRoleForDataset() ([]bq.AccessRole, error) {
+	var roles = []bq.AccessRole{bq.OwnerRole, bq.WriterRole, bq.ReaderRole}
+	var resourceName string
+	ctx := context.Background()
+	datasetIterator := c.client.Datasets(ctx)
+	for {
+		dataset, err := datasetIterator.Next()
+		if err == iterator.Done || err != nil {
```

Comment:
This one `iterator` or `datasetIterator` ?

Reply:
this one datasetIterator

Reply:
should this be fixed?

Reply:
fixed